consensus/src/execution_pipeline.rs

```diff
@@ -111,8 +112,9 @@ impl ExecutionPipeline {
             parent_block_id,
             block_preparer,
             result_tx,
+            command_creation_time,
         } = command;
-
+        counters::PREPARE_BLOCK_WAIT_TIME.observe_duration(command_creation_time.elapsed());
```

Comment:
While you are at it, can you add some counter around time it takes to verify the signature?

Reply:
Done.